src/code-templates/services/order-service/Dockerfile

```diff
@@ -1,6 +1,26 @@
-FROM node:14.2.0-alpine3.11
+FROM node:16 AS dependencies
 WORKDIR /app
-COPY package.json /app/
-COPY package-lock.json /app/
-COPY package-lock.json /app/
-RUN npm ci --only=production
+COPY package*.json .
+RUN npm install
+
+FROM dependencies AS build
```

Comment:
Why not unify the install and build phase? To my understanding, the concept of stages meant for cases where you must start with something 'big' and then peak only subset to the next phase. In this case, the dep phase copies everything from the former phase, so why not simplify and unify?